### DISCUSSION TITLE
Can this app do a slide show of photos?

### DISCUSSION THREAD

**Discussion by balajeek** [photoprism/photoprism#3127] [General]
As the title says, wondering if there are options where it can run a fullscreen photo slide show on the browser?

**Comment by lastzero** [comment]
Open a picture by clicking on it, then press the play button at the top.

**Comment by balajeek** [comment]
Ah! that easy, i did not even realize that.  Thanks.
I have a plan to keep my home server monitor always on which will show photo slide show on the top and some grafana dashboard on the bottom, using a html page with iframe.  hope i can succeed with that and its not high in terms of resource uses.  
Thanks.
